ballerina/utils.bal

```diff
@@ -244,6 +244,18 @@ isolated function populateCreateObjectHeaders(map<string> requestHeaders, Object
     }
 }
 
+# Function to populate createObject optional user-defined metadata headers.
+#
+# + requestHeaders - Request headers map.
+# + userMetadataHeaders - Map containing user-defined metadata.
+isolated function populateUserMetadataHeaders(map<string> requestHeaders, map<string> userMetadataHeaders){
+    if(userMetadataHeaders.length() != 0) {
```

Comment:
We don't need to check the length.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-aws.s3/pull/109/commits/764284a9697f1ef980cbfcfe0223c33693f80279